src/main/java/slimeknights/tconstruct/library/modifiers/hook/ranged/BowAmmoModifierHook.java

```diff
@@ -100,21 +100,21 @@ private static ItemStack findMatchingAmmo(ItemStack bow, LivingEntity living, Pr
    * @param tool       Tool instance
    * @param bow        Bow stack instance
    * @param predicate  Predicate for valid ammo
-   * @param player     Player to search
+   * @param user       User to search
    * @return  Found ammo
    */
-  static ItemStack findAmmo(IToolStackView tool, ItemStack bow, Player player, Predicate<ItemStack> predicate) {
+  static ItemStack findAmmo(IToolStackView tool, ItemStack bow, LivingEntity user, Predicate<ItemStack> predicate) {
```

Comment:
Usually I call the entity using a tool `holder`, since it makes their role more clear in the grand scheme of tool usage.

Reply:
Completed